eva/utils/profiler.py

```diff
@@ -42,10 +54,22 @@ def run(self):
         Param: None
 
         Returns: List[Metrics]
-        """
-        # TODO: Implement the actual logic
-        # Use self._classobj's methods to run for various batch sizes
+        """        
+
+        #NOTE This wont work if input_format is -1. Need to implement for that case.
+
+        expected_h = self._classobj.input_format.height
+        expected_w = self._classobj.input_format.width
+        expected_c = self._classobj.input_format.channels
+
+        batch_sizes = [1,10,100,1000] 
+        metrics_list=[]
+        for id, batch in enumerate(batch_sizes): # 1 5 25 125
+            input_tensor = torch.rand(batch, expected_c, expected_w, expected_h).float()
```

Comment:
Is float intentional? Aren't these pixel values & thus in decimals? 

float64 operations take much longer than decimal operations. Is the float() intentional? If so could you please share the reason?

Reply:
Had to use float because I was running into some issues when running running the tensor through the model. I can check if it works with decimal and update the PR